.github/workflows/build.yml

```diff
@@ -109,41 +76,33 @@ jobs:
         uses: ocaml/setup-ocaml@v3
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
-          dune-cache: true
-          opam-pin: false
-
-      - run: opam install conf-pkg-config
```

Comment:
We need this, this is a temporary fix for https://github.com/dra27/mingw-w64-shims/issues/1